README.md

```diff
@@ -1,2 +1,195 @@
-# oni-lda
-LDA parallel version for OpenNetworkInsight
+# oni-lda-c
+
+Parallel implementation of latent Dirchlet allocation for OpenNetworkInsight, version 1.0.1
+
+oni-lda-c is a program for *topic modelling*:  From a collection documents and integer k, infer k topics and assign to each document a mixture of topics and to each topic a mixture of words. 
+
+Topic modelling is used in the OpenNetworkInsights suspicious connections analysis to generate profiles of typical traffic on the network and estimate
+the probability of a given communication occurring between two IPs. A pair of IPs is a "document"
+and the communications between the IPs are the "words". The "topics" are then profiles common traffic on the network, the mixture of topic to the documents
+is a decomposition of the traffic between two IPs into these common traffic profiles.  For given communication, we can estimate its probablity of occurrence between a given IP pair and flag it as "suspicious".
+
+To learn more about topic modelling, we suggest that you start with [these materials](https://www.cs.princeton.edu/~blei/topicmodeling.html).
+
+**On the shoulders of giants**: oni-lda-c is an MPI parallelization of [the BleiLab's single processor lda-c program](https://github.com/blei-lab/lda-c).
+
+## Getting Started
+
+These instructions are for installing oni-lda-c as a standalone topic modelling tool. If you wish to install oni-lda-c as part of OpenNetworkInsights, please follow the instructions [here for version 1.0.1](https://github.com/Open-Network-Insight/open-network-insight/wiki). 
+Even if you wish only to use oni-lda-c as a part of oni, the material in this README file may be helpful for configuring and understanding oni-lda-c.
+
+### Prerequisites
+
+oni-lda-c requires an MPI installation on the cluster. It has been validated with [mpich](http://www.mpich.org/) and [intel-mpi](https://software.intel.com/en-us/intel-mpi-library). You must install your favorite MPI on your cluster following the instructions provided by that
+distribution.
+
+### Compilation
+
+Compilation requires that MPI libraries have been installed on the machine where oni-lda-c will be compiled. Other than that, just make in the top oni-lda-c directory:
+```
+[user@system ~]$ cd oni-lda-c
+[user@system  oni-lda-c]$ make
```

Comment:
Current wiki indicates to run make clean before make, is make clean not required anymore? 
https://github.com/Open-Network-Insight/open-network-insight/wiki/Install%20and%20Configure%20ML